packages/node-sdk/README.md

```diff
@@ -127,14 +127,14 @@ Note: BUCKET_FEATURES_ENABLED, BUCKET_FEATURES_DISABLED are comma separated list
 
 ```json
 {
-  secretKey: "...",
-  logLevel: "warn",
-  offline: true,
-  host: "https://proxy.slick-demo.com"
-  featureOverrides: {
-    huddles: true,
-    voiceChat: false
-  },
+  "secretKey": "...",
+  "logLevel": "warn",
+  "offline": true,
+  "host": "https://proxy.slick-demo.com",
+  "featureOverrides": {
```

Comment:
`featureOverrides` is declared as follows:
```
featureOverrides?:
    | string
    | ((context: Context) => Partial<Record<keyof TypedFeatures, boolean>>);
```
so you should use an arrow function

Reply:
this is the config file which takes a simple boolean map

Reply:
Ah!